### ISSUE TITLE
Avoid installing deprecated type packages

### ISSUE THREAD

**Issue by silverwind** [jeffijoe/typesync#114]
According to https://github.com/jeffijoe/typesync/issues/79, deprecated type packages should not be suggested or installed, but I notice it still does, for example with this package.json:

```json
{
  "devDependencies": {
    "escape-goat": "4.0.0",
    "escape-string-regexp": "5.0.0"
  }
}
```

```
$ npx typesync --dry
»  TypeSync v0.12.1
»  —— DRY RUN — will not modify file ——
✔  2 new typings can be added.

📦 test — package.json (2 new typings added)
├─ + @types/escape-string-regexp
└─ + @types/escape-goat

✨  Run typesync again without the --dry flag to update your package.json.
```

Both type packages are deprecated as seen during `npm install`:

```
npm warn deprecated @types/escape-goat@2.0.1: This is a stub types definition. escape-goat provides its own type definitions, so you do not need this installed.
npm warn deprecated @types/escape-string-regexp@2.0.1: This is a stub types definition. escape-string-regexp provides its own type definitions, so you do not need this installed.
```

**Comment by jeffijoe** [comment]
I think I know what the issue is. When there's no matching version of the typings package, it'll fall back to checking the first one in the list of versions. The code assumes that to be the latest version, but NPM seems to be returning them in the other order.

**Comment by jeffijoe** [comment]
Fixed in v0.12.2. Thanks for reporting!

**Comment by silverwind** [comment]
Thanks, working with 0.12.2!

```
$ npx typesync@0.12.2 --dry
»  TypeSync v0.12.2
»  —— DRY RUN — will not modify file ——
✔  No new typings to add, looks like you're all synced up!
